### PR TITLE
[8.0] workflow task & Workflow activity action

### PR DESCRIPTION
**Pull request by adrienpeiffer**
- [x] Depends #110

### CODE REVIEW

**Comment by sbidoul** [comment]
Can you write some more documentation in the READMEs?

**Comment by adrienpeiffer** [comment]
@sbidoul. It's done. Thanks !

**Comment by adrienpeiffer** [comment]
Rebased

File: workflow_task/README.rst [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+    :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+    :alt: License: AGPL-3
+
+==============
+Workflow tasks
+==============
+
+This module automatically creates tasks when entering workflow activities.
+
+In addition to workflow_activity_acion module, this one allow you to create tasks when an object enters on an activity.
```
Comment (sbidoul) [inline workflow_task/README.rst:11]: s/acion/action/

File: workflow_task/README.rst [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+
+Tasks must be configured on the activity concerned. It is possible to define a Deathline based on a date field and a critical delay during which the task is displayed in red.
```
Comment (sbidoul) [inline workflow_task/README.rst:13]: deadline suffira, ne soyons pas morbides :wink:

Comment (adrienpeiffer) [inline workflow_task/README.rst:13]: oups :joy:

**Comment by adrienpeiffer** [comment]
rebased

**Comment by sbidoul** [comment]
Closing old stuff.

**Pull request closed without merging**